k2/csrc/ragged_ops.h

```diff
@@ -388,10 +400,17 @@ Ragged<T> Transpose(Ragged<T> &src,
                            same value
       @param [in] num_srcs Number of source shapes to append; require
                            num_srcs > 0.
+      @param [out] merge_map  If not nullptr, will be set to the merge-map
+                          that tells us for each 0 <= i < ans.NumElements(),
+                          which element of `src` it came from (available
+                          as `ans[i]i % num_srcs`) and its element-index within
+                          `src[i]` (available as `ans[i] / num_srcs`.
+
       @param [in] src      Array of sources to append
```

Comment:
this line should be in front of the documentation of `merge_map` according to the parameter order.